public/index.html

```diff
@@ -36,7 +36,8 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
 
-    <link rel="preconnect" href="%REACT_APP_AMPLITUDE_PROXY_URL%" />
```

Comment:
why remove amplitude though?

Reply:
nevermind, i see that the proxy live at the same domain+subdomain as the quote API

Reply:
maybe comment that is intended to catch  quotes, proxies etc under the umbrella